bot.py

```diff
@@ -33,6 +34,12 @@ def setup():
                                  'prefix',
                                  'COBALT_key'])
 
+    logger = logging.getLogger('discord')
+    logger.setLevel(logging.DEBUG)
+    handler = logging.FileHandler(filename='logfile.log', encoding='utf-8', mode='w')
```

Comment:
For the sake of running a single bot on multiple servers, perhaps this should be `server name.log` instead

Reply:
Currently the bot doesn't support running multiple connections simultaneously since `client.run()` actually blocks. If we want to be able to support multiple servers we should probably restructure slightly.